lib/providers/linkedin.js

```diff
@@ -0,0 +1,39 @@
+// Load modules
+
+var Crypto = require('crypto');
+
+// Declare internals
+
+var internals = {};
+
+
+exports = module.exports = function (options) {
+    
+    return {
+        protocol: 'oauth2',
+        auth: 'https://www.linkedin.com/uas/oauth2/authorization',
+        token: 'https://www.linkedin.com/uas/oauth2/accessToken',
+        scope: ['r_fullprofile', 'r_emailaddress', 'r_contactinfo'],
+        scopeSeparator: ',',
+        profile: function (credentials, params, get, callback) {
+            var query = {
```

Comment:
we have a different style guide.  I'll update the linting rules at some point... but please add a newline after every function declaration.  The reason is to make it easier to identify new scopes